test/network_autoyast_test.rb

```diff
@@ -319,4 +319,33 @@ def keep_install_network_value(value)
       end
     end
   end
+
+  describe "#valid_rename_udev_rule?" do
+    it "fails when the rule do not contain new name" do
+      rule = { "rule" => "exists", "value" => "as_well" }
```

Comment:
well the bug was about missing attributes ;-) but ok, I'll use these meaningful values.